server/sonar-web/src/main/js/apps/coding-rules/facets/repository-facet.js

```diff
@@ -44,7 +44,7 @@ export default CustomValuesFacet.extend({
 
   getLabelsSource() {
     const source = {};
-    this.options.app.repositories.forEach(repo => source[repo.key] = repo.name);
+    this.options.app.repositories.forEach(repo => (source[repo.key] = repo.name));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Extract the assignment of "source[repo.key]" from this expression. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AAssignmentWithinCondition)